src/pages/StatisticsChart/activities/ContractResourceDistributed.tsx

```diff
@@ -0,0 +1,105 @@
+import { useTranslation } from 'react-i18next'
+import { DATA_ZOOM_CONFIG } from '../../../utils/chart'
+import { SmartChartPage } from '../common'
+import { ChartItem, explorerService } from '../../../services/ExplorerService'
+import { ChartColorConfig } from '../../../constants/common'
+import config from '../../../config'
+
+const useOption = (
+  statisticContractResourceDistributed: ChartItem.ContractResourceDistributed[],
+  chartColor: ChartColorConfig,
+  isMobile: boolean,
+  isThumbnail = false,
+): echarts.EChartOption => {
+  const { t } = useTranslation()
+  const gridThumbnail = {
+    left: '4%',
+    right: '10%',
+    top: '8%',
+    bottom: '6%',
+    containLabel: true,
+  }
+  const grid = {
+    left: '3%',
+    right: '3%',
+    top: '8%',
+    bottom: '5%',
+    containLabel: true,
+  }
+  return {
+    color: chartColor.colors,
+    grid: isThumbnail ? gridThumbnail : grid,
+    dataZoom: isThumbnail ? [] : DATA_ZOOM_CONFIG,
+    xAxis: [
+      {
+        name: isMobile || isThumbnail ? '' : t('statistic.address_count'),
+        nameLocation: 'middle',
+        nameGap: 30,
+        type: 'log',
+        splitLine: { show: false },
+      },
+    ],
+    yAxis: [
+      {
+        type: 'log',
+        splitLine: { show: false },
+        name: isMobile || isThumbnail ? '' : t('statistic.ckb_amount'),
+      },
+    ],
+    tooltip: {
+      position: 'top',
+      enterable: true,
+      formatter: (params: any) => {
+        return `<a href=https://${config.BASE_URL}/script/${params.value[3]}/${params.value[5]} target=_blank>${
```

Comment:
Can React only jump to script route, ignore domain?

Reply:
https://github.com/Magickbase/ckb-explorer-frontend/blob/develop/src/pages/StatisticsChart/activities/AddressBalanceRank.tsx#L117-L127
It helps, works as https://ckb-explorer-frontend-in-magickbase-repo-git-7cb256-magickbase.vercel.app/charts/address-balance-rank